### PR TITLE
Result page

### PR DESCRIPTION
**Pull request by Maianki**
## Added result page 

Result page screenshot

## Live Link
https://result-page-preview.netlify.app/quiz-pages/guessplayer-quiz/result.html

### CODE REVIEW

File: quiz-pages/guessplayer-quiz/result.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            <img class="img-result" src="../../assets/result-img.svg" alt="winning poster">
+            
+            <h2 class="quiz-score-heading">Your Score: <span class="quiz-score text-xl">19 / 25</span></h2>
+
+           <section class="quiz-answers flex-row">
+
+                    <!--QUESTION - 1-->
+                <div class="card card-question">
+                    <div class="card-header">     
+                        <img class="card-img card-horizontal-img responsive-img" src="../../assets/saina nehwal.jpg" alt="">
```
Comment (vrinda-mahajan) [inline quiz-pages/guessplayer-quiz/result.html]: I think you forget to add the value for alt attribute here.

File: quiz-pages/guessplayer-quiz/result.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+           </section>
+        
+           <a href="../../index.html#categories" class="btn btn-primary text-center" role="button">Back to Quizees</a>
+        </main>
+
+         <footer class="footer flex-column align-items-center">
+            <p class="text-sm">Made with <span class="footer-credit"><i class="fas fa-heart"></i> </span>by <span class="footer-credit"> Ankit Kumain</span></p>
+            <ul class="flex-row footer-links">
+                <li class="list-unstyled">
+                    <a class="nav-link md-ht-1 footer-menu-link" href="https://github.com/Maianki"><i class="fab fa-github-square"></i></a>
```
Comment (vrinda-mahajan) [inline quiz-pages/guessplayer-quiz/result.html]: You can add target="_blank" to the anchor tag. It will open your social media links in a new tab.

**Review by vrinda-mahajan — APPROVED** [review]
The code looks perfect💯. Just added a few suggestions.

File: quiz-pages/guessplayer-quiz/result.html (hunk 1 above)
Comment (Maianki) [reply quiz-pages/guessplayer-quiz/result.html]: Thanks for pointing that out, will fix it.

File: quiz-pages/guessplayer-quiz/result.html (hunk 2 above)
Comment (Maianki) [reply quiz-pages/guessplayer-quiz/result.html]: Okay I will add that.

**Review by MayankKumar10 — APPROVED** [review]
Looks good to me you can merge it.

**Comment by Maianki** [comment]
> Looks good to me you can merge it.

Thanks for reviewing.

**Review by Anurag021 — APPROVED** [review]
Looks good to me.

**Comment by Maianki** [comment]
> Looks good to me.

Thanks for reviewing.

**Pull request merged**